File: python3/iniciante/1160.py
def calculate_years(pa, pb, g1, g2):
    anos = 0
    while anos <= 100 and pa <= pb:
        anos += 1
        pa += int(pa * (g1 / 100))
        pb += int(pb * (g2 / 100))

    if anos > 100:
        return "Mais de 1 seculo."
    else:
        return str(anos) + " anos."

n = int(input())

for _ in range(n):
    pa, pb, g1, g2 = map(float, input().split())
    result = calculate_years(pa, pb, g1, g2)
    print(result)

File: c/iniciante/1160.c
#include <stdio.h>
#include <math.h>

int main() {
    int n, anos;
    double pa, pb, g1, g2;

    scanf("%d", &n);

    for (int i = 0; i < n; i++) {
        scanf("%lf %lf %lf %lf", &pa, &pb, &g1, &g2);

        anos = 0;
        while (1) {
            if (anos > 100 || pa > pb) {
                break;
            }

            anos += 1;
            pa += trunc(pa * (g1 / 100));
            pb += trunc(pb * (g2 / 100));
        }

        if (anos > 100) {
            printf("Mais de 1 seculo.\n");
            continue;
        }

        printf("%d anos.\n", anos);
    }

    return 0;
}
